ci/otel.sh

```diff
@@ -38,6 +38,6 @@ mkdir build
 cd build
 
 cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_TESTING=OFF -DWITH_OTLP=ON -DWITH_OTLP_HTTP=ON -DWITH_OTLP_GRPC=OFF
```

Comment:
The compilation prints deprecation warning

```
CMake Warning at CMakeLists.txt:165 (message):
  WITH_OTLP is deprecated and will be removed in future.  Please set either
  WITH_OTLP_GRPC or WITH_OTLP_HTTP, or even both.
```

https://github.com/apache/qpid-proton/actions/runs/5089577302/jobs/9147379969#step:10:205

Reply:
@kpvdr IIRC you had trouble packaging the GRPC part, yes? Maybe it's an option to ignore the gRPC functionality, like it is done here? (Just thinking aloud, but it would make things simpler if it was the case.)

Reply:
Its setting WITH_OTLP,  and also setting both WITH_OTLP_HTTP + WITH_OTLP_GRPC...the warnings seems like the latter two are replacement options, so can we just remove the first one now?

Reply:
Just to follow up here: Looks like we can't remove any of the flags for now, as it is leading to otel build failure.